src/Swarm/Game/Scenario/Objective.hs

```diff
@@ -113,7 +113,7 @@ objectiveAchievement :: Lens' Objective (Maybe AchievementInfo)
 instance FromJSON Objective where
   parseJSON = withObject "objective" $ \v ->
     Objective
-      <$> (fmap . map) reflow (v .:? "goal" .!= [])
+      <$> (v .:? "goal" .!= [])
```

Comment:
When I tried to use `Node` instead of `[Node]` no errors were shown in integration tests.

I think it is because of laziness - a list always evaluates to `Node Document`.

@byorgey how should I get around it? I suppose something like `deepseq` could work or I could make the entire `Node` strict. :thinking:

Reply:
I'm not sure I understand the problem.  What do you mean by "use `Node` instead of `[Node]`"?

Reply:
The whole description should be one document that we render.

Right now it is a list of documents, even though markdown supports paragraphs.

Reply:
Using `Node` only should parse the document as JSON array (see the ToJSON instance that uses Semigroup)

Reply:
OK, I see now why we want to use `Node` instead of `[Node]`, and why that should work.  And just to make sure I'm on the same page, right now if you build the code as is, we get a bunch of errors, as we should.  But you're saying that if you use `Node` instead of `[Node]` it doesn't generate any errors?  I don't understand what laziness would have to do with that.

Reply:
Maybe the error is not evaluated, but it may also be that I have my parser function wrong. :shrug: 

I will try to get back to this tomorrow.

Reply:
I simplified the types and this problem went away. 🤷